myhpi/settings.py

```diff
@@ -156,7 +156,7 @@
 
 STATIC_URL = env.str("STATIC_URL")
 STATIC_ROOT = env.str("STATIC_ROOT")
-STATICFILES_DIRS = (os.path.join(BASE_DIR, "static"),)
+STATICFILES_DIRS = (os.path.join(BASE_DIR, "myhpi/static"),)
```

Comment:
@jeriox Do you have a clue why I needed to add this in order to find the static files with `python manage.py collectstatic`?

Reply:
oh yeah it was just the wrong path in the settings file. my fault, didn't change that when we refactored stuff a while ago. It's not necessary to collect the static files for local development, therefore this did not stand out. thx for the catch